### PR TITLE
2684 - simplify long alt text for images

### PR DESCRIPTION
**Pull request by christineyost**
This PR is for [#2684](https://app.zenhub.com/workspaces/event-bus-team-632b50a9a7bf8400277f2cf8/issues/gh/department-of-veterans-affairs/ves/2684). It simplifies long alt text for images on the developer portal. In some instances alt text was simply reworded, and in others text from the alt descriptions was moved to the main content of the page.

### CODE REVIEW

**Review by lsmitchell — COMMENTED** [review]
I like moving some of the hidden info into the main body, but if I couldn't read the image (using a screen reader, etc) I'd be much more satisfied with the old description to let me know exactly what the image was. I think we'd be better served with the more literal description alt texts, imo.

Edit: I see the point of the ticket was to lower the alt text lengths, but there is a lot going on in the diagrams, particularly the first one. Maybe keep the shortened descriptions and just say "as described above" or something to reference the text that's been expanded?

File: docs/intro-to-eda.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ title: Event-Driven Architecture at VA
...
+_The following conceptual diagram illustrates how producers and consumers might interact with the Enterprise Event Bus. Producers publish many different kinds of events. Consumers may do many different things with event data, such as notify a Veteran or kick off a workflow. [View the full-sized diagram (must be part of VA GitHub organization to view).](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Diagrams/future%20state%20whole.png)_
 
-![A conceptual diagram showing various event-producing systems on the left, which funnel into business topics in the center, which are finally filtered out to consuming systems on the right.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/f0dfe62a-8509-459c-bd9a-074e0babb22b)
+![The conceptual diagram illustrating producers publishing various events to Event Bus and consumers using the event data for various tasks.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/f0dfe62a-8509-459c-bd9a-074e0babb22b)
```
Comment (alwerner) [inline docs/intro-to-eda.md:13]:
```suggestion
![A conceptual diagram illustrating how producers publish various events to the Event Bus and how consumers may use the event data for different purposes.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/f0dfe62a-8509-459c-bd9a-074e0babb22b)
```

File: docs/intro-to-eda.md [hunk 2]
```diff
@@ -43,10 +43,9 @@ When determining what information an event should contain, a guiding principle i
...
 
-The Enterprise Event Bus uses [AWS Managed Streams for Kafka (MSK)](https://docs.aws.amazon.com/msk/), a hosted version of Kafka that runs in the VA Enterprise Cloud. 
+The Enterprise Event Bus uses [AWS Managed Streams for Kafka (MSK)](https://docs.aws.amazon.com/msk/), a hosted version of Kafka that runs in the VA Enterprise Cloud. Event-producing systems publish events to Kafka topics that are available on AWS MSK. Events in a single topic are distributed across different brokers and partitions to balance load. Metadata managed by the cluster informs producing and consuming systems which broker they need to connect to. The data on AWS MSK is stored in [AWS Elastic Block Store (EBS)](https://docs.aws.amazon.com/ebs/). Event-consuming systems can subscribe to topics to pull streams of events. The diagram below shows how events are stored in AWS MSK and how producers and consumers interact with the system.
```
Comment (alwerner) [inline docs/intro-to-eda.md:46]:
```suggestion
The Enterprise Event Bus uses [AWS Managed Streams for Kafka (MSK)](https://docs.aws.amazon.com/msk/), a hosted version of Kafka that runs in the VA Enterprise Cloud. Producer systems publish events to Kafka topics that are available on AWS MSK. Events in a single topic are distributed across multiple brokers and partitions in order to balance load. Metadata managed by the cluster informs producing and consuming systems which broker they need to connect to. The data on AWS MSK is stored in [AWS Elastic Block Store (EBS)](https://docs.aws.amazon.com/ebs/). Consumer systems can subscribe to topics to pull streams of events. The diagram below shows how events are stored in AWS MSK and how events are distributed from producers to consumers via AWS MSK.
```

File: docs/intro-to-eda.md [hunk 3]
```diff
@@ -43,10 +43,9 @@ When determining what information an event should contain, a guiding principle i
...
-![A diagram showing a producer application on the left, pushing a stream of events to a topic. In the center of the diagram, events are shown in a box called AWS MSK (indicating that they are hosted there) and distributed among brokers and partitions to balance the load. There is a note explaining that metadata managed by the cluster informs producers and consumers which broker to connect to. The data on AWS MSK is stored in AWS EBS. Consumer applications can subscribe to topics to pull streams of events, as the consuming application is doing in the right side of the diagram.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)
+![The diagram showing the architecture of AWS Managed Streams for Kafka and how producers and consumers interact with it, as described above.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)
```
Comment (alwerner) [inline docs/intro-to-eda.md:48]:
```suggestion
![Diagram showing a high-level overview of AWS MSK and how events are distributed from producers to consumers using multiple brokers in AWS MSK.](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)
```

**Review by alwerner — APPROVED** [review]
Approved with suggestions.

**Pull request merged**